src/components/projectDetail/ProjectTabs.tsx

```
@@ -194,44 +212,55 @@ const ProjectTabs = ({ projectData, isLoading, error }: ProjectTabsProps) => {
           {/* 탭 메뉴 */}
-          <div className="border-b border-gray-border">
-            <div className="flex space-x-8">
-              {["소개", "프로젝트 정보", "후기"].map((tab) => (
+          <div className="border-b border-gray-border relative">
+            <div className="flex space-x-8 relative">
+              {tabList.map((tab, idx) => (
                 <button
                   key={tab}
-                  className={`relative pb-3 text-xl font-medium ${activeTab === tab ? "text-main-color" : "text-sub-gray"}`}
+                  ref={el => { tabRefs.current[idx] = el; }}
+                  className={`relative pb-4 text-xl font-medium ${activeTab === tab ? "text-main-color" : "text-sub-gray"}`}
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)